src/winetricks

```diff
@@ -11682,6 +11724,49 @@ load_foobar2000()
 
 #----------------------------------------------------------------
 
+w_metadata hhw apps \
+    title="HTML Help Workshop" \
+    publisher="Microsoft" \
+    year="2000" \
+    media="download" \
+    file1="htmlhelp.exe" \
+    installed_exe1="$W_PROGRAMS_X86_WIN/HTML Help Workshop/hhw.exe"
+
+load_hhw()
+{
+    w_call mfc40
+
+    # https://msdn.microsoft.com/en-us/library/windows/desktop/ms669985(v=vs.85).aspx
+    w_download http://download.microsoft.com/download/0/A/9/0A939EF6-E31C-430F-A3DF-DFAE7960D564/htmlhelp.exe b2b3140d42a818870c1ab13c1c7b8d4536f22bd994fa90aade89729a6009a3ae
+
+    # hhupd.exe shows a dialog that says
+    # "This computer already has a newer version of HTML Help."
```

Comment:
Hm, wine doesn't have htmlhelp afaict. Does using a lower windows version help, maybe?

Reply:
Interestingly, setting Windows version to "Windows 95" doesn't help (the dialog appears). "Windows NT 3.51" doesn't work as well. Older versions (e.g. "Windows 3.1") cannot be used because the installer says "Windows 95 or Windows NT is required to install".

Reply:
Winxp?

Reply:
Setting Windows version to "Windows XP" doesn't help.
* 10, 8.1, 8, 2008 R2, 7, 2008, Vista, 2003, XP, 2000, ME, 98, 95, NT 4.0, NT 3.51 -> "This computer already has a newer version of HTML Help." 
* 3.1, 3.0, 2.0 -> "Windows 95 or Windows NT is required to install"

`hhupd.exe` works when:
* `hhctrl.ocx` doesn't exist in `$W_SYSTEM32_DLLS`
* and `hhctrl.ocx` is disabled (e.g. `WINEDLLOVERRIDES="hhctrl.ocx="`)

However there are 2 problems:
* `hhupd.exe` works only when Windows version is "Windows 98", "Windows 95", "Windows NT 4.0", or "Windows NT 3.51" (can not specified in 64-bit prefixes), otherwise it only shows an error dialog ("HTML Help is a Windows system component and can only be updated via a service pack on this version of Windows.")
* The installer dialog ("This computer does not have HTML Help installed. Click OK to install HTML Help 1.3.") appears even if `/Q` option is specified

In the end I decided to edit `htmlhelp.inf` to avoid running `hhupd.exe`.

Reply:
Thanks for the details (and double thanks for documenting it in the source!)